addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -424,9 +424,11 @@ export class MassMailingHtmlField extends HtmlField {
         // Hide or show the help message if some templates are visible
```

Comment:
Can you update this comment ? (so we are 100% precise)

Reply:
Sure :)
Made it "Hide or show the help message and preview wrapper based on whether there are any relevant templates"